WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderPostUiStateBuilder.kt

```diff
@@ -167,9 +167,10 @@ class ReaderPostUiStateBuilder @Inject constructor(
 
     fun mapPostToBlogSectionUiState(
         post: ReaderPost,
+        isReaderImprovementsEnabled: Boolean,
         onBlogSectionClicked: (Long, Long) -> Unit
     ): ReaderBlogSectionUiState {
-        return buildBlogSection(post, onBlogSectionClicked)
+        return buildBlogSection(post, onBlogSectionClicked, isReaderImprovementsEnabled = isReaderImprovementsEnabled)
```

Comment:
Nitpick: since the name of the parameter `isReaderImprovementsEnabled` is the same in `mapPostToBlogSectionUiState` and `buildBlogSection`, would it make sense to not make it a named argument?

e.g. 
```
return buildBlogSection(post, onBlogSectionClicked, isReaderImprovementsEnabled)
```

Reply:
This is actually needed because `buildBlogSection` has other arguments, so in order to be able to only pass the `isReaderImprovements` argument while still using the defaults for the others I need to use it as named argument.

https://github.com/wordpress-mobile/WordPress-Android/blob/17e604c6f8a7590f0f207c690500b03ba92c7e60/WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderPostUiStateBuilder.kt#L230-L236